zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -99,8 +99,14 @@ public static void main(String[] args) throws Exception {
     jettyServer.setHandler(contexts);
 
     LOG.info("Start zeppelin server");
-    jettyServer.start();
-    LOG.info("Started");
+    try {
+      jettyServer.start();
+    } catch (Exception e) {
+      LOG.error("Error while running jettyServer", e);
+      //TODO(jl): Fix the error status code
```

Comment:
Do you plan to submit a patch for this PR soon? 
As TODOs are for some changes that your are going to do soon, otherwise i.e for me or for any other reader of this comment, It's not obvious how exactly it supposes to be fixed and why right now it is broken.